resource_estimator/src/system/data/report.rs

```diff
@@ -36,12 +36,14 @@ impl Report {
         formatted_counts: &FormattedPhysicalResourceCounts,
     ) -> Self {
         let logical_counts = result.layout_overhead();
+        let part = result.factory_parts()[0].as_ref();
```

Comment:
Dispalying just the first one? Why?

Reply:
The system instantiation only supports T states as magic states, but the generic implementation allows to model systems with more than one magic state type.

Reply:
I added a comment.